base/asyncmap.jl

```diff
@@ -260,13 +260,6 @@ function asyncmap(f, b::BitArray; kwargs...)
     return b2
 end
 
-# TODO: Optimize for sparse arrays
-# For now process as regular arrays and convert back
-function asyncmap(f, s::AbstractSparseArray...; kwargs...)
```

Comment:
I deprecated this in `stdlib/SparseArrays/src/deprecated.jl` since it is better to be transparent to the user that we don't have an optimized method for this anyway.